examples/image_classification_kornia.ipynb

```diff
@@ -0,0 +1,3856 @@
+{
```

Comment:
<p>It'd be great to add a Trainer subsection/heading over here. I don't think most people are aware of this functionality</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/226/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>